cpp/URLHostObjects.cpp

```diff
@@ -199,58 +249,124 @@ namespace fasturl {
     };
 
     std::vector<jsi::PropNameID> URLHostObject::getPropertyNames(
-        jsi::Runtime &rt) {
+        jsi::Runtime &rt)
+    {
         std::vector<jsi::PropNameID> keys;
-        const char *names[] = {"href",     "origin", "protocol", "username",
-                               "password", "host",   "hostname", "port",
-                               "pathname", "search", "hash",     "toString"};
-        for (const auto &name : names) {
+        const char *names[] = {"href", "origin", "protocol", "username",
+                               "password", "host", "hostname", "port",
+                               "pathname", "search", "hash"};
+        for (const auto &name : names)
+        {
             keys.push_back(jsi::PropNameID::forAscii(rt, name));
         }
         return keys;
     };
 
     jsi::Value URLHostObject::get(jsi::Runtime &rt,
-                                  const jsi::PropNameID &propNameID) {
+                                  const jsi::PropNameID &propNameID)
+    {
         std::string key = propNameID.utf8(rt);
-        if (key == "href") {
+        if (key == "href")
+        {
             return HOST_STR(std::string(_url.get_href()));
         }
-        if (key == "origin") {
+        if (key == "origin")
+        {
             return HOST_STR(std::string(_url.get_origin()));
         }
-        if (key == "protocol") {
+        if (key == "protocol")
+        {
             return HOST_STR(std::string(_url.get_protocol()));
         }
-        if (key == "username") {
+        if (key == "username")
+        {
             return HOST_STR(std::string(_url.get_username()));
         }
-        if (key == "password") {
+        if (key == "password")
+        {
             return HOST_STR(std::string(_url.get_password()));
         }
-        if (key == "host") {
+        if (key == "host")
+        {
             return HOST_STR(std::string(_url.get_host()));
         }
-        if (key == "hostname") {
+        if (key == "hostname")
+        {
             return HOST_STR(std::string(_url.get_hostname()));
         }
-        if (key == "port") {
+        if (key == "port")
+        {
             return HOST_STR(std::string(_url.get_port()));
         }
-        if (key == "pathname") {
+        if (key == "pathname")
+        {
             return HOST_STR(std::string(_url.get_pathname()));
         }
-        if (key == "search") {
+        if (key == "search")
+        {
             return HOST_STR(std::string(_url.get_search()));
         }
-        if (key == "hash") {
+        if (key == "hash")
+        {
             return HOST_STR(std::string(_url.get_hash()));
         }
-        if (key == "toString") {
```

Comment:
`toString` should just return `href` (same as `toJSON`) according to [here](https://developer.mozilla.org/en-US/docs/Web/API/URL/toString)